hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestMetaRegionReplicaReplicationEndpoint.java

```diff
@@ -338,4 +476,134 @@ static boolean contains(List<Result> contains, List<Cell> cells) throws IOExcept
     }
     return !containsScanner.advance() && matches >= 1 && count >= matches && count == cells.size();
   }
+
+  private void doNGets(final Table table, final byte[][] keys) throws Exception {
```

Comment:
nit: for helper methods who's purpose is to make assertions, I like to prefix their names with `assert`. So, `assertDoNGets`, `assertPrimaryNoChangeReplicaIncrease`, &c.